app/src/main/java/com/github/featuredetectandroid/MainActivity.kt

```diff
@@ -115,7 +86,7 @@ class MainActivity : ComponentActivity() {
     }
```

Comment:
is it correct to call super before your call here?

Reply:
fixed